### PR TITLE
#11084: Update ternary bw docs

### LINKED ISSUE #11084: Update documentation for Ternary bw ops
Tracking Issue : #10585

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
Tracking Issue : #10585, Issue : #11084
https://github.com/tenstorrent/tt-metal/actions/runs/10266847620 - passed

### CODE REVIEW

File: ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward_pybind.hpp [hunk 1]
```diff
@@ -20,7 +20,7 @@ namespace ternary_backward {
 namespace detail {
 
 template <typename ternary_backward_operation_t>
-void bind_ternary_backward(py::module& module, const ternary_backward_operation_t& operation, const std::string& description) {
+void bind_ternary_backward(py::module& module, const ternary_backward_operation_t& operation, const std::string& description, const std::string& supported_dtype) {
     auto doc = fmt::format(
```
Comment (eyonland) [inline ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward_pybind.hpp:24]: Again, another magic enum?

File: ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward_pybind.hpp [hunk 2]
```diff
@@ -20,7 +20,7 @@ namespace ternary_backward {
 namespace detail {
 
 template <typename ternary_backward_operation_t>
-void bind_ternary_backward(py::module& module, const ternary_backward_operation_t& operation, const std::string& description) {
+void bind_ternary_backward(py::module& module, const ternary_backward_operation_t& operation, const std::string& description, const std::string& supported_dtype) {
```
Comment (patrickroberts) [inline ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward_pybind.hpp:23]: Please change all the
```cpp
const std::string& description, const std::string& supported_dtype
```
to
```cpp
std::string_view description, std::string_view supported_dtype
```

**Review by eyonland — APPROVED** [review]
Approving based under assumption you will update based on @patrickroberts

**Review by patrickroberts — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/10277957615 - passed

**Pull request merged**